downloads/platform.md

```diff
@@ -25,7 +25,21 @@ Uninstall Julia by deleting Julia.app and the packages directory in ~/.julia. Mu
 
 ## Linux
 
-It is strongly recommended that the official generic linux binaries from the downloads page be used to install Julia. The following distribution-specific packages are community contributed. They may not use the right versions of Julia dependencies or include important patches that the official binaries ship with. In general, bug reports will only be accepted if they are reproducible on the generic linux binaries on the downloads page.
+It is strongly recommended that the official generic linux binaries from the downloads page be used to install Julia. 
+
+### Generic Linux Binaries
+
+The generic linux binaries do not require any special installation steps, but you will need to ensure that your system can find the `julia` executable. To do that, you will need to extract the `.tar.gz` file downloaded from the [downloads page](index.html) to a folder on your computer, then either create a symbolic link to `julia` from a folder which is on your system `PATH` or add the `/bin` folder inside that archive to your system's `PATH` environment variable.
```

Comment:
Good points! I've updated the wording to address this. Could you please take a look?